core/LSU.cpp

```diff
@@ -1199,27 +1201,29 @@ namespace olympia
     // Update issue priority after store instruction retires
     void LSU::updateIssuePriorityAfterStoreInstRetire_(const InstPtr & inst_ptr)
     {
-        for (auto & inst_info_ptr : ldst_inst_queue_)
-        {
-            if (inst_info_ptr->getInstPtr() == inst_ptr)
+        if(!inst_ptr->isVector()){
```

Comment:
Same comment as above. Just assert if vector since vector insts should never end up here.